site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -65,9 +65,16 @@ Functions which changes GameServer state or settings are:
 1. SetLabel()
 1. SetAnnotation()
 1. Allocate()
-1. Reserve()
+1. Reserve() 
+{{% feature publishVersion="1.6.0" %}}
```

Comment:
The feature gate here made some weird spacing:

<img width="512" alt="Screen Shot 2020-05-18 at 12 35 29 AM" src="https://user-images.githubusercontent.com/7751204/82186308-7c3fcf00-989f-11ea-9d1d-6a802b320b96.png">

I'm thinking this will get fixed when you cut 1.6 and remove the extra markup though, so maybe not worth fixing?

Reply:
There is one more option to duplicate items from 1 to 6 and nest them `expiryVersion` shortcode.

Reply:
Yeah, I went with the easy option, since it will only show in the dev docs right now, and will get fixed as soon as we cut to 1.6.